File: CS502018/pset1/mario/more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //counter and variables
    int height = 0;
    int counter = 1;

    //ask user for height
    do
    {
        height = get_int("Height: ");
    }
    while(height > 23);

    while(height != 0)
    {
        //adds whitespace before #
        for(int j = 0; j < (height - 1); j++)
        {
            printf(" ");
        }

        //prints first set of #
        for(int i = 0; i < counter; i++)
        {
            printf("#");
        }

        //prints whitespace that seperate the #
        printf("  ");

        //prints the second set of #
        for(int i = 0; i < counter; i++)
        {
            printf("#");
        }

        //next line
        printf("\n");

        //decrease/increase variables
        height--;
        counter++;
    }
    return 0;
}

File: CS502017/pset6/mario.py
import cs50

def main():

    print("Height:", end="")

    height = get_height()

    for i in range(height):

        print(" " * (height - 1), end ="")

        print("#" * (i + 2), end="")

        print("")

        height = height - 1

def get_height():

    while True:
        n = cs50.get_int()

        if n > 0 and n <= 23:
            break

    return n

if __name__ == "__main__":
    main()